### ISSUE TITLE
getting 'ModuleNotFoundError: No module named 'yaml' when running manager.py 

### ISSUE THREAD

**Issue by ace510** [mikeyy/omeglebot#3]
File "manager.py", line 10, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'

probably pretty simple, just need to know what library you're using to manage your YAML data, AFAIK it's not in the standard library

**Comment by ace510** [comment]
i'm guessing PyYaml, but I want to be sure

**Comment by mikeyy** [comment]
You are right, it's pyyaml. I will pack this up into a nice distribution package at some point. I was waiting on adding requirements.txt before I did that but I will go ahead and add that although I think that might be the only one, hah.

**Comment by mikeyy** [comment]
There we go, I've added a requirements.txt file so you can do `pip install -r requirements.txt`
